package.json

```diff
@@ -51,7 +51,8 @@
 		"type-check": "tsc --noEmit"
 	},
 	"dependencies": {
-		"@typescript-eslint/utils": "^5.62.0"
+		"@typescript-eslint/scope-manager": "^6.21.0",
```

Comment:
this doesn't seem to be directly used anywhere?

Reply:
It is in `lib/node-utils/index.ts` as `@typescript-eslint/utils` doesn't export `TSESLintScope` anymore as of v6